Sazabi.cpp

```diff
@@ -927,6 +949,10 @@ void CSazabi::ParseSingleParam(CString param) {
 		{
 			m_strOptionParam = trimedParam;
 		}
+		else if (paramValue.Find(_T("chronosconfig")) == 0)
+		{
+			m_strConfigParam = trimedParam;

```

Comment:
```suggestion
			m_strConfigParam = trimmedParam;

```

いまさらだけどtrimedがあちこちにあるのが気になる